compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DataflowAnalyzer.java

```diff
@@ -604,6 +613,17 @@ public void visit(BLangInvocation invocationExpr) {
         }
     }
 
+    private boolean isValidSelf(BSymbol symbol) {
+
+        for (BField field : ((BObjectType) symbol.type).fields) {
+            InitStatus initStatus = this.uninitializedVars.get(field.symbol);
+            if (initStatus != null) {
+                return false;
+            }
```

Comment:
Can we use `this.uninitializedVars.containsKey(field.symbol)` instead of this?

Reply:
Changed.